src/openzaak/components/autorisaties/models.py

```diff
@@ -63,7 +63,7 @@ class Meta:
         # informatieobjecttypen/besluittypen, the scopes/VA are constant for
         # these. The exact *type is implied by the component choice, so we need
         # a unique constraint on applicatie + component
-        unique_together = ("applicatie", "component")
+        unique_together = ("applicatie", "component", "scopes")
```

Comment:
Is this recommend? It feels wrong.

Reply:
As far as I know, the form should maintain the order of the scopes. I have added a test for this.

Reply:
@Coperh it is possible to use array field in unique index in postgres, but yeah it needs either presorting or adding custom Postgres `sort` function and creating index with the usage of this function.